fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_SelectWptPage.js

```diff
@@ -39,8 +39,8 @@ class A320_Neo_CDU_SelectWptPage {
                     freq = " ";
                 }
 
-                const latString = (w.infos.coordinates.lat.toFixed(0) >= 0) ? `${w.infos.coordinates.lat.toFixed(0).toString().padStart(2, "0")}N` : `${Math.abs(w.infos.coordinates.lat.toFixed(0)).toString().padStart(2, "0")}S`;
-                const longString = (w.infos.coordinates.long.toFixed(0) >= 0) ? `${w.infos.coordinates.long.toFixed(0).toString().padStart(3, "0")}E` : `${Math.abs(w.infos.coordinates.long.toFixed(0)).toString().padStart(3, "0")}W`;
+                const latString = `${Math.abs(w.infos.coordinates.lat).toFixed(0).toString().padStart(2, "0")}${w.infos.coordinates.lat >= 0 ? 'N' : 'S'}`;
+                const longString = `${Math.abs(w.infos.coordinates.long).toFixed(0).toString().padStart(3, "0")}${w.infos.coordinates.long >= 0 ? 'E' : 'W'}`;
```

Comment:
I don't think `.toString()` is necessary as `.toFixed()` already converts to a string. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
Oh, I didn’t even notice that. I was more concerned with the other bits 😂.